Packages/MIES/MIES_SweepFormula.ipf

```diff
@@ -437,3 +523,75 @@ Function/WAVE GetSweepForFormula(graph, rangeStart, rangeEnd, channelType, sweep
 
 	return sweeps
 End
+
+Function setvar_sweepFormula_update(sva) : SetVariableControl
+	STRUCT WMSetVariableAction &sva
+
+	String mainPanel, yFormula, xFormula
+	Variable numFormulae
+
+	switch(sva.eventCode)
+		case 1: // mouse up
+		case 2: // Enter key
+		case 3: // Live update
+			String sval = sva.sval
+			mainPanel = GetMainWindow(sva.win)
+			DFREF dfr = BSP_GetFolder(mainPanel, MIES_BSP_PANEL_FOLDER)
+			NVAR/Z status = dfr:sweepFormulaParse
+			if(!NVAR_EXISTS(status))
+				DebugPrint("Global variable sweepFormulaParse not found")
+				return 0
+			endif
+			status = 0
+			SVAR/Z formula = dfr:sweepFormula
+			if(!SVAR_EXISTS(formula))
+				DebugPrint("Global variable sweepFormula not found. Can not evaluate parsing status.")
+				return 0
+			endif
+
+			SplitString/E="^(.+?)(?:\s+vs\s+(.+))?$" formula, yFormula, xFormula
+			numFormulae = V_flag
+			if(numFormulae != 2 && numFormulae != 1)
+				DebugPrint("Display command must follow the \"y[ vs x]\" pattern. Can not evaluate parsing status.")
+				return 0
+			endif
+
+			try
+				JSON_Release(FormulaParser(yFormula))
+				status = 1
+				if(numFormulae == 1)
+					return 0
+				endif
+				DebugPrint("y part of formula is valid.")
+				JSON_Release(FormulaParser(xFormula))
+			catch
+				status = 0
+			endtry
+			break
+		case -1: // control being killed
+			break
+	endswitch
+
+	return 0
+End
+
+Function button_sweepFormula_display(ba) : ButtonControl
```

Comment:
maybe button name and procedure name could be named differently to see if that produces the conflict but another solution could be that the procedure file "MIES_SweepFormula.ipf" is not included in the "MIES_include" file